velox/dwio/common/exception/Exceptions.cpp

```diff
@@ -54,8 +54,14 @@ void verify(bool c, std::string fmt...) {
 
 std::string error_string(std::string fmt, va_list ap) {
   std::array<char, 1024> buf;
```

Comment:
let's initialize 0 here since this is not performance-critical.

Reply:
Done.